packages/runtime/src/client/index.ts

```diff
@@ -354,7 +354,9 @@ export class MakeswiftClient {
       )
     }
 
-    this.graphqlClient = new GraphQLClient(new URL('graphql', apiOrigin).href)
+    this.graphqlClient = new GraphQLClient(new URL('graphql', apiOrigin).href, {
+      'Makeswift-Runtime-Version': PACKAGE_VERSION,
```

Comment:
Just as a side note, we should probably switch to lowercase header naming convention going forward.